lib/helpers/commands.js

```diff
@@ -0,0 +1,306 @@
+'use babel';
+import Logger from './logger.js'
+import ApiWrapper from '../wrappers/api-wrapper.js';
+const pretty = require('prettysize');
+
+export default class Commands  {
+
+  constructor(device,pyboard,terminal,settings){
+    this.device = device
+    this.pyboard = pyboard
+    this.terminal = terminal
+    this.settings = settings
+    this.api = new ApiWrapper(settings)
+    this.logger = new Logger('commands')
+  }
+
+  prepare(cb){
+    var _this = this
+    this.pyboard.stop_running_programs(function(err){
+      if(err) return err
+      _this.pyboard.enter_raw_repl_no_reset(function(err){
+        cb(err)
+      })
+    })
+  }
+
+  exit(cb){
+    var _this = this
+    _this.pyboard.enter_friendly_repl(function(err){
+      cb(err)
+    })
+  }
+
+  getAllBoardInfo(cb){
+    var info = {}
+    var _this = this
+    this.prepare(function(err){
+      if(err) return err
+      _this.getWifiApName(function(content){
+        info['wifi_ap_name'] = content
+        _this.getMac(function(content){
+          info['mac'] = content
+          _this.getFreeMemory(function(content){
+            info['free_memory'] = content
+            _this.getFreeRam(function(content){
+              info['free_ram'] = content
+              _this.getVersion(function(content){
+                info['version'] = content
+                _this.getPartition(function(content){
+                  info['partition'] = content
+                  _this.getFSType(function(content){
+                    info['fs_type'] = content
+                    _this.getWifiIP(function(content){
+                      info['wifi_ip'] = content
+                      _this.getWifiMode(function(content){
+                        info['wifi_mode'] = content
+                        _this.getPartition(function(content){
+                          info['partition'] = content
+                          _this.getBoardType(function(content){
+                            info['board_type'] = content
+                            _this.getLoRaMac(function(content){
+                              info['lora_mac'] = content
```

Comment:
Please rewrite this callback with async/await 
You might need to install and use babel to be able to use async/await

Reply:
Ah okay. I thought I should merge first so then I could implement eslint.

Reply:
It's fine to apply eslint now, because pymakr-2.0 branch is not behind the development branch now (so eslint fix shouldn't bring any conflicts).